src/main/java/com/google/sps/data/DBUtils.java

```diff
@@ -8,11 +8,15 @@
 import java.util.concurrent.ExecutionException;
 
 public class DBUtils {
-  private static final String DB_COMMENTS = "comment-collection";
+  public static final String DB_COMMENTS = "comment-collection";
+  public static final String RECIPE_CONTENT_KEY = "content";
```

Comment:
I think that `RECIPE_CONTENT_KEY` and `TAG_IDS_KEY` belong in their respective obj. classes rather than here in DBUtils

Reply:
Also I think CONTENT_KEY makes more sense as an identifier once it's moved to the respective classes

Reply:
Agreed, done in db71017.